libpod/container_inspect.go

```diff
@@ -48,6 +48,11 @@ func (c *Container) getContainerInspectData(size bool, driverData *inspect.Data)
 		hostnamePath = getPath
 	}
 
+	exitCode := int32(0)
+	if runtimeInfo.State == ContainerStateStopped {
```

Comment:
I think `docker inspect` will report the exit code from the last time the container was run

Reply:
Hm. It doesn't - seems to reset back to 0 every time the container is started.

Reply:
I don't think podman inspect should show an exit code from the previous run.